src/core/schema/pandoc-output-formats.ts

```diff
@@ -0,0 +1,71 @@
+/*
+* pandoc-output-formats.ts
+*
+* Copyright (C) 2021 by RStudio, PBC
+*
+*/
+
+// FIXME this should be autogenerated from $ pandoc --list-output-formats
+export const pandocOutputFormats = [
```

Comment:
I'll fix manually and add a post-merge TODO to generate this from actually running `pandoc --list-output-formats`. We can do it in `quarto build-js`, which is now needed for everything anyway. Incidentally, we probably ought to rename the command to `quarto build-prerequisites` since we also build some JSON...